### PR TITLE
Added FilesystemLoader Test

### PR DESCRIPTION
**Pull request by peterrehm**
Regarding #166.

I added Tests for the Filesystem Loader based on `Symfony\Bundle\TwigBundle\Tests\Loader\FilesystemLoaderTest`.

### CODE REVIEW

File: Tests/Twig/Loader/FilesystemLoaderTest.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+use Liip\ThemeBundle\ActiveTheme;
+use Liip\ThemeBundle\Twig\Loader\FilesystemLoader;
+use Symfony\Bundle\FrameworkBundle\Templating\TemplateReference;
+
+class FilesystemLoaderTest extends \PHPUnit_Framework_TestCase
+{
+    public function testGetSourceContextWithLocator()
+    {
+        $activeTheme = new ActiveTheme('test', ['test']);
```
Comment (lsmith77) [inline Tests/Twig/Loader/FilesystemLoaderTest.php]: feel free to drop PHP 5.3 and 5.4 from the test matrix .. or adjust to use the old array syntax

**Comment by peterrehm** [comment]
I have updated the test matrix.

**Comment by peterrehm** [comment]
@lsmith77 I changed 5.3 with prefer-lowest to 7.0. That is the only test which fails. I am not sure why...

**Comment by lsmith77** [comment]
I guess we have to review the deps that get installed for lowest and check if we should bump some min requirement to something more recent ..

**Comment by lsmith77** [comment]
will have a look tomorrow ..

**Comment by lsmith77** [comment]
looks like the min version of twig needs to be bumped to 1.27 for the tests .. 

lets see https://github.com/liip/LiipThemeBundle/pull/168

**Comment by peterrehm** [comment]
Awesome, thanks!

**Pull request merged**